Source/WebCore/platform/graphics/ColorConversion.h

```diff
@@ -61,25 +61,73 @@ template<typename Output, typename Input> inline Output convertColor(const Input
     return ColorConversion<CanonicalColorType<Output>, CanonicalColorType<Input>>::convert(color);
 }
 
+// MARK: White Point.
+
+inline constexpr float const D50WhitePoint[] = { 0.3457 / 0.3585, 1.0, (1.0 - 0.3457 - 0.3585) / 0.3585 };
+inline constexpr float const D65WhitePoint[] = { 0.3127 / 0.3290, 1.0, (1.0 - 0.3127 - 0.3290) / 0.3290 };
```

Comment:
What does the `inline` do here?

We don't need the `const` and it should be omitted.